test/integration/055_ref_override_test/test_ref_override.py

```diff
@@ -0,0 +1,26 @@
+from test.integration.base import DBTIntegrationTest, use_profile
+
+
+class TestRefOverride(DBTIntegrationTest):
+    @property
+    def schema(self):
+        return "dbt_ref_override_055"
+
+    @property
+    def project_config(self):
+        return {
+            'data-paths': ['data'],
+            "macro-paths": ["macros"],
+        }

```

Comment:
you probably need a `'quote_columns': False,` entry here too, to quiet that deprecation warning.